src/main/java/seedu/toluist/controller/TaskController.java

```diff
@@ -110,4 +110,8 @@ private CommandResult delete(TodoList todoList, Task task) {
         todoList.remove(task);
         return new CommandResult(DELETE_TASK_RESULT);
     }
+
+    public static String[] getCommandWords() {
+        return new String[] { UPDATE_TASK_COMMAND, DELETE_TASK_COMMAND, ADD_TASK_COMMAND };
```

Comment:
oh i thought you mean the three words UPDATE_TASK_COMMAND, DELETE_TASK_COMMAND, ADD_TASK_COMMAND need to be placed in order (so ADD_TASK_COMMAND, DELETE_TASK_COMMAND then UPDATE_TASK_COMMAND)

to address the actual comment, it should follow the English order. In this case you say update task command (ie. command to update task), not command to update task